pkg/kubelet/qos/policy.go

```diff
@@ -72,8 +72,8 @@ func GetContainerOOMScoreAdjust(pod *v1.Pod, container *v1.Container, memoryCapa
 		return (1000 + guaranteedOOMScoreAdj)
 	}
 	// Give burstable pods a higher chance of survival over besteffort pods.
-	if int(oomScoreAdjust) == besteffortOOMScoreAdj {
-		return int(oomScoreAdjust - 1)
+	if int(oomScoreAdjust) >= besteffortOOMScoreAdj {
+		return int(besteffortOOMScoreAdj - 1)
```

Comment:
Thanks for the review! `besteffortOOMScoreAdj` is a const value, however it needs to change its value to add a bad test case. So I do not addd test case for it, is it OK?